### PR TITLE
fix #61973 digitalcitizen.life

### LINKED ISSUE #61973: digitalcitizen.life
### Issue URL (Ads)
 [https://www.digitalcitizen.life/how-reset-all-your-windows-10-network-adapters-just-6-clicks](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.digitalcitizen.life%2Fhow-reset-all-your-windows-10-network-adapters-just-6-clicks)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5
Browser: | Edge 84.0.522.61 (Chromium-based)
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (180), <br/>Block WebRTC, <br/>Block Push API, <br/>Block Location API, <br/>Block Flash
Filters: | <b>Ad Blocking:</b><br/> Peter Lowe's Blocklist, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> I don't care about cookies, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Malware Domains Blocklist, <br/>Spam404<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>Polskie Filtry Ciasteczkowe, <br/>KAD - Przekręty, <br/>Polish Anti Adblock Filters, <br/>AlleBlock, <br/>EasyList Polish
Custom filter subscriptions: | https://gitlab.com/ZeroDot1/CoinBlockerLists/raw/master/list_browser_UBO.txt,<br>https://raw.githubusercontent.com/PolishFiltersTeam/PolishAnnoyanceFilters/master/PAF_pop-ups.txt,<br>https://raw.githubusercontent.com/PolishFiltersTeam/KAD/master/KADfake.txt

### PR DESCRIPTION
**Pull request by stanislav-atr**
#61973

### CODE REVIEW

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]
Please block also this newsletter (without social links of course)
Screenshot 2020-08-22 at 21 17 53

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -271,6 +271,7 @@ cbc.ca##.ad-risingstar-container
 bonusbitcoin.co##div[style="text-align:center;margin-bottom:10px;height:300px"]
 hidden247.com##.oglSvePoc
 lolalytics.com###leaderboard
+digitalcitizen.life##aside[id*="-amp-ad-"]
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt:274]: why not `block--amp-ad`?

**Comment by stanislav-atr** [comment]
> 
> 
> Please block also this newsletter (without social links of course)
> Screenshot 2020-08-22 at 21 17 53

There is identical widget at the footer of the page, should we maybe delete this one completely?

File: AnnoyancesFilter/sections/annoyances.txt [hunk 2]
```diff
@@ -81,6 +81,8 @@ belaruspartisan.by##.news-detail a[href="https://t.me/belaruspartisanby"]
 mywebpc.ru#?#.sidebar_list > div.widget:has(> div.textwidget > a[href="tg://resolve?domain=mywebpc"])
 ! https://github.com/AdguardTeam/AdguardFilters/issues/55890
 sankei.com###snsFollow
+! https://github.com/AdguardTeam/AdguardFilters/issues/61973
+digitalcitizen.life##.block-next-sidebar-subscription-options > :not(.subscription-options__social_buttons)
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:85]: for subscriptions we have the section
https://github.com/AdguardTeam/AdguardFilters/blob/c4bbe243c52e19d72f7bdee07be6529658f225ea/AnnoyancesFilter/sections/annoyances.txt#L1476

**Review by Alex-302 — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**